runtime/gc_glue_java/PointerArrayObjectScanner.hpp

```diff
@@ -112,7 +113,7 @@ class GC_PointerArrayObjectScanner : public GC_IndexableObjectScanner
 		return objectScanner;
 	}
 
-	MMINLINE uintptr_t getBytesRemaining() { return sizeof(fomrobject_t) * (_endPtr - _scanPtr); }
+	MMINLINE uintptr_t getBytesRemaining() { return _elementSize * (_endPtr - _scanPtr); }
```

Comment:
This change is correct but would it be cleaner just do `((uintptr_t)_endPtr - (uintptr_t)_scanPtr)` ?

Reply:
Obviously current implementation returns value aligned to `_elementSize` and suggested does not necessarily. I am not sure do we use this fact anywhere (pass unaligned addresses). So would be safer to keep current implementation

Reply:
I'm still tempted to do the fast thing here (I did it elsewhere already). If the pointers aren't aligned to _elementSize, then the pointer math is going to be undefined anyway.

Reply:
this is correct